#goal
int64 max_numberf
float64 wait_duration
---
#result
int64 count
---
#feedback
float64 percentage